README.markdown

```diff
@@ -293,7 +293,7 @@ Deletes all instances of the undef value from an array or hash. For example, `$h
 
 Returns the difference between two arrays. The returned array is a copy of the original array, removing any items that also appear in the second array. For example, `difference(["a","b","c"],["b","c","d"])` returns ["a"]. *Type*: rvalue.
 
-#### `dig`
+#### `dig44`
```

Comment:
Please keep the entry for the old dig() around and replace its description with the deprecation warning to reduce possible confusion.